benches/gen_benchmark_inputs.py

```diff
@@ -24,23 +24,6 @@ def gen_field_summate(path, seed):
     np.savetxt(path / 'field_bench_z_1.txt', rm._z_1)
     np.savetxt(path / 'field_bench_z_2.txt', rm._z_2)
 
-    pos_no = 100000
-    mode_no = 1000
-    x = np.linspace(0.0, 10.0, pos_no)
-    y = np.linspace(-5.0, 5.0, pos_no)
-    z = np.linspace(-6.0, 8.0, pos_no)
-
-    model = gs.Gaussian(dim=3, var=1.0, len_scale=1.0)
-
-    rm = RandMeth(model, mode_no, seed)
-
-    np.savetxt(path / 'field_vs_x.txt', x)
-    np.savetxt(path / 'field_vs_y.txt', y)
-    np.savetxt(path / 'field_vs_z.txt', z)
-    np.savetxt(path / 'field_vs_cov_samples.txt', rm._cov_sample)
-    np.savetxt(path / 'field_vs_z_1.txt', rm._z_1)
-    np.savetxt(path / 'field_vs_z_2.txt', rm._z_2)
-
```

Comment:
I use these files for comparing the Rust implementations with the Cython implementations. I feel, that the comparison Python script does not belong to this repo, but for development, it might be useful to share it. I will add it to the repository soon and delete it later. As soon as all Rust implementations are comparable to Cython execution speed.

Reply:
Depending on the answer to the original question concerning problem sizes, should I just write the large data sets under both file names? Revert the change and benchmark with the smaller problems again?

Reply:
(Or probably rather switch the benchmarks to use the `*_vs_*` files and drop the `*_bench_*` variants.)

Reply:
It's probably a good idea to remove the `*_bench_*` files completely and and rename the `*_vs_*` files to `*_*`. Are you willing to do this? - Otherwise, just keep both files for the moment and I'll remove the `*_vs_*` files in a later commit.

Reply:
Changed as suggested.